TicketBookingBackend/TicketBookingAPI/TicketBookingAPI/Controllers/TicketBookingController.cs

```diff
@@ -8,17 +8,48 @@ namespace TicketBookingAPI.Controllers
     [ApiController]
     public class TicketBookingController : Controller
     {
-        private CityService _service;
-        public TicketBookingController(CityService service)
+        private CityService _cityservice;
+        private BusService _busService;
+        private TicketService _ticketService;
+        public TicketBookingController(CityService cityService,BusService busService, TicketService ticketService)
         {
-            _service = service;
+            _cityservice = cityService;
+            _busService = busService;
+            _ticketService = ticketService;
         }
 
         [HttpGet("/{pattern}")]
         public async Task<ActionResult<IEnumerable<CityModel>>> GetAllCities(string pattern)
         {
-            var cities = await _service.GetAllCitiesLike(pattern);
+            var cities = await _cityservice.GetAllCitiesLike(pattern);
             return Ok(cities);
         }
+
+        [HttpPost]
```

Comment:
why it is a post method ?

Reply:
we are sending request with model in body.